src/components/charts/common/chartDatumUtils.ts

```diff
@@ -342,58 +345,33 @@ export function getDateRange(
   return [start, end];
 }
 
+// returns the abbreviated month name (MMM format)
+export function getAbbreviatedMonth(year, month) {
+  return intl.formatDate(new Date(year, month), { month: 'short' });
```

Comment:
This is only used for the purpose of getting the abbreviated month , we are not relying on it for any dates.
It replaces the need to have this in the messages 
  "months_abbr": [
    "Jan",
    "Feb",
    "Mar",
    "Apr",
    "May",
    "Jun",
    "Jul",
    "Aug",
    "Sep",
    "Oct",
    "Nov",
    "Dec"
  ],

Reply:
It's mainly for the sake of consistency, since we have to use `format` everywhere else

Reply:
I moved back to the date-fns style using the locale option. we will need to figure out hw we can dynamically import locales based on the application local being set for this to work properly .

Example using xdate-fns to handle abbreviated month name and locale

`
const locale = require('date-fns/locale/en-US'); // This could be messy

// returns the abbreviated month name (MMM format)
export function getAbbreviatedMonth(year, month) {
  return format(new Date(year, month), 'MMM', { locale });
}
`